packages/strapi-admin/services/permission/permissions-manager/index.js

```diff
@@ -30,10 +26,19 @@ module.exports = ({ ability, action, model }) => ({
     return this.sanitize(data, { ...options, isOutput: false });
   },
 
-  queryFrom(query) {
+  getQuery(queryAction = action) {
+    if (_.isUndefined(queryAction)) {
```

Comment:
As it was discussed elsewhere, shouldn't we prefer the fp method over the regular one?